docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -231,6 +231,13 @@ test
 
 If the start page is not specified, it defaults to `about:blank`.
 
+To test webpages in local directories, you can use the `file://` protocol.
+
+```js
+fixture `MyFixture`
```

Comment:
Oh, you speaking about `file:` protocol, your example shows URL. If so, yes, it would be nice to have relative path support (I suppose we don't have one at the moment)